completions/bash/multipass

```diff
@@ -227,6 +242,9 @@ _multipass_complete()
             _multipass_aliases
             opts="${opts} ${multipass_aliases}"
         ;;
+        "get")
+            opts="${opts} --raw"
```

Comment:
Ideally, this would only be added if not already used. But we can probably let it stick, as that's what we're doing for the likes of `--help` already. And it is not technically wrong to say `--raw` multiple times.

Reply:
Fixed with

```suggestion
            if [[ ! ${COMP_WORDS[*]} =~ --raw ]]; then
                opts="${opts} --raw"
            fi
```